### ISSUE TITLE
Optimize get_Option in pintable.cpp to be more user friendly by not requiring anything but option name

### ISSUE THREAD

**Issue by Lovinity** [vpinball/vpinball#1803]
**Is your feature request related to a problem? Please describe.**
It is very unintuitive to have to specify minValue, maxValue, step, unit, and variants every time you use Table.Option as a getter. If you decide to change this at any time, you have to go through the entire script and change it everywhere you use that option. Furthermore, when implementing a new option, you have to specify these everywhere in the script.

Furthermore, it's code smell because get_Option is almost a duplicate of put_Option.

**Describe the solution you'd like**
I would like the Option variable of Tables to operate as follows:

* You must first register a setting by calling Table.Option as a setter (ideally in Table_Init). There, you specify all the parameters.
* After registering a setting, you can access it at any time using Table1.Option("option name"); no other args are necessary, just option name. If the option has not yet been registered, the script will error saying so.

**Describe alternatives you've considered**
I could use constants to hold the values of all the args and then put the consts at the top of the script (so all I'd have to change is the value of the constant), but that would be code smell. And constants do not support arrays (for the array of literal strings)). Dims would resolve that but that's even worse code smell since those are not intended to ever change after being defined.

**Comment by Lovinity** [comment]
An effective but not very elegant workaround is using a Scripting.Dictionary object and assigning Table1.Option calls to values.

e.g. Dictionary.Item("Option Name") = Table1.Option("Option Name", ...)
...and then accessing the value using Dictionary.Item("Option Name").

Necessitates that all the Dictionary items are reassigned when options are changed via OptionEvent so they are updated with the new option values. I do this on event 3.

**Comment by vbousquet** [comment]
In fact, it's defined like this fully on purpose: 
- it avoids having a 2 step with stored state system, leading to safer code. With the current design an option is always valid against the definition given when it is accessed. If a table author wants to do some state management, it is on his side. VPX will always hold to the definition it is given when it is get or put.
- It allows to have a minimal API made of a single line command for managing options.

**Comment by Lovinity** [comment]
I don't agree it is safer code. On VPX's end, sure it is safer. But it significantly compromises code safety on the author's end. Given they have to define the validation parameters every time they use the call, that opens up a whole can of worms of them forgetting to change something somewhere in the code (or making a typo).

And this isn't elegant to work around either. Constants don't allow arrays. So you'd need a huge mess of constants if you want to manage validation with constants (one constant per parameter per option). The best way is to have them defined in a Sub and assigned to a Dictionary, and then ensuring the Sub is called both on Init and on OptionEvent. It works (and ensures the Table.Option is only ever defined once per option in the code, thus reducing the risk of typos and bugs), but it isn't elegant nor is it safe.
